cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectAssemblyOperatorMockTest.java

```diff
@@ -117,13 +122,18 @@ public void afterEach() {
 
         Promise created = Promise.promise();
 
+        Deployment depBeforeRecon = mockClient.apps().deployments().inNamespace(NAMESPACE).withName(KafkaConnectResources.deploymentName(CLUSTER_NAME)).get();
```

Comment:
If you call `createConnectCluster` and the cluster is paused ... how come it already exists? I would expect that if I create the KafkaConnect CR already with the paused annotation nothing will happen and no deployment will exist.